docs/catalog/cloud/match-slo.md

```diff
@@ -0,0 +1,39 @@
+---
+version: 1.0
+submitted_by: danielmeixner
+published_date: TBD
+category: cloud
+tags: 
+ - Cloud
+ - Reliability/Availability
+ - Monitoring
+---
```

Comment:
Please add a complexity tag.  See more here- https://github.com/Green-Software-Foundation/green-software-patterns/blob/dev/docs/guide/suggested-tags.md#complexity. I would say this pattern has a complexity of small.

Reply:
done.

Reply:
I am struggling to see the complexity tag. Would you mind taking another look? here is an example- https://github.com/Green-Software-Foundation/green-software-patterns/blob/dev/docs/catalog/cloud/match-utilization-requirements-with-pre-configured-server.md?plain=1#L9